camayoc/tests/qpc/yupana/test_connect.py

```diff
@@ -0,0 +1,41 @@
+"""Starting tests for configs"""
+
+import requests
+
+from camayoc.tests.qpc.yupana.utils import (
+    yupana_configs,
+    get_app_url,
+    post_file,
+    get_x_rh_identity,
+)
+
+
+EXPECTED_CONFIGS = ("app_name", "openshift_project", "openshift_app_domain", "api_path")
+
+
+def test_config():
+    """test load configurations"""
+    config = yupana_configs()
+    assert config != []
```

Comment:
Provide some feedback to the assertion in case of an error: `assert config != [], "Empty configuration!"`.

Reply:
Pytest will provide the message saying that config was expected to be different than an empty list, so adding the message seems redundant to me

Reply:
Yea, I think I'll leave this one because the default pytest return is more useful than if I say something.